Automattic/wp-calypso
# Split `lib/analytics/ad-tracking`

This PR splits up `lib/analytics/ad-tracking` from one massive module into multiple modules with an index file.

This should help modularise analytics, so that not everything needs to get loaded upfront.

There are no code changes other than code moving around, and always calling `setupGtag()` on setup.

#### Changes proposed in this Pull Request

* Split `lib/analytics/ad-tracking` into modules

#### Testing instructions

There are little to no code changes, so it should be sufficient to ensure that there are no build or unit test failures, which should pick up a missing or incorrectly named export. For extra peace of mind, a quick check to make sure that ad tracking functionality continues to work correctly could be useful.
